internal/http/services/owncloud/ocdav/copy.go

```diff
@@ -293,16 +294,26 @@ func (s *svc) descend(ctx context.Context, client gateway.GatewayAPIClient, src
 			return fmt.Errorf("status code %d", httpDownloadRes.StatusCode)
 		}
 
+		fileName, fd, err := s.createChunkTempFile()
```

Comment:
see below ... this copies the file to disk before sending it out again ... just read from the body